x/oracle/keeper/test_utils.go

```diff
@@ -285,8 +285,8 @@ func AllocateRewards(t *testing.T, input TestFixture, rewards sdk.Coins, votePer
 }
 
 var (
-	stakingAmt         = sdk.TokensFromConsensusPower(10, sdk.DefaultPowerReduction)
-	randomExchangeRate = sdk.NewDec(1700)
+	TEST_STAKING_AMT = sdk.TokensFromConsensusPower(10, sdk.DefaultPowerReduction)
```

Comment:
Let's avoid global variables in tests.

Reply:
It's just a rename to make it more obvious that we're referencing global vars. Should we just add them as test variables to the struct?

Reply:
Or is it sufficient to just lowercase the `T` to make it private?

Reply:
It's more about not creating globally mutable variables. It's a bad practice because it creates the possibility of accidental mutation and which could have unknown downstream consequences.

Reply:
Test variables are only available in the package_test namespace, so they can't be used in the normal business logic